app/views/static_pages/home.html.erb

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Sample App</title>
+  </head>
+  <body style="overflow-y: scroll; overflow-x: hidden; display: block; margin: 0px;">
```

Comment:
-  Trình bày lại convention toàn bộ đoạn này quá lộn xộn
- Chuyển style vào file scss k nhúng trực tiếp trong html như thế này